task/09-functions-n-closures-tasks.js

```diff
@@ -150,7 +171,11 @@ function logger(func, logFunc) {
  *   partialUsingArguments(fn, 'a','b','c','d')() => 'abcd'
  */
 function partialUsingArguments(fn) {
-    throw new Error('Not implemented');
+    let arg = Array.from(arguments).filter((e, i) => i != 0)
```

Comment:
most interesting approach with `rest parameters` can be used here to avoid  `filter`